README.md

```diff
@@ -1,3 +1,18 @@
 # otp-admin-ui
 
 Front end application to manage OpenTripPlanner and [otp-middleware](https://github.com/ibi-group/otp-middleware).
+
+## Getting started
+
+```
+git clone https://github.com/ibi-group/otp-admin-ui.git
+cd otp-admin-ui
+cp next.config.tmp.js next.config.js
+# Update next.config.js values
```

Comment:
These line is outdate, please update with the procedure regarding the new .env file.